src/commands/client-kill.json

```diff
@@ -45,7 +45,8 @@
                 "type": "oneof",
                 "arguments": [
                     {
-                        "name": "ip:port",
+                        "name": "old-format",
+                        "display": "ip:port",
```

Comment:
I feel like "display_name" would be more descriptive for what this is. Display just seems incomplete.

Reply:
i'm ok with display_name
@oranagra @itamarhaber ?

Reply:
i'm not sure why adding "name" to "display" makes it better.
in many cases, like the one quoted here, it's a display string, or a suggestion.
so maybe `display_text` is the right thing.

Reply:
@itamarhaber please vote

Reply:
I like plain `display`. I can live with `display_name`. How about `nickname`?

Reply:
`nickname` feels odd

let's stay with `display`?

Reply:
I'm okay with display_text or display_name. I just have no association with a standalone "display". I don't like nickname. If I had had to rank:
`display_name`, `display_text`, `display`, `nickname`.

Reply:
i don't see a problem with `display`, and i don't like `display_name` (not sure why it's a **name**).
so let's go with `display_text`